pkg/log/eventlog/eventlog_windows.go

```diff
@@ -33,7 +34,19 @@ type eventLogger struct {
 func (l *eventLogger) Log(keyvals ...interface{}) error {
 	lb := l.getLoggerBuf()
 	defer l.putLoggerBuf(lb)
-	if err := lb.logger.Log(keyvals...); err != nil {
+
+	// fmtlogger does not support array, chan, func, slice, struct, or map
+	// so we'll do any pre-processing for these types here
```

Comment:
Seems okay. though I _also_ none of this should ever land in the event logs, so I don't know if we need it

Reply:
They will currently land in the event logs -- do we not want them to?